set-up.R

```diff
@@ -53,33 +53,17 @@ remove_style = function(x){
 
 # function to add model_output_table class to all tables
 add_table_class <- function(x){
-  
   x <- gsub("<table>", "<table class='model_output_table'>", x)
   x
 }
 
-save_formats <- function(to_save, name = F, csv = F){
+save_formats <- function(to_save, name = F){
   if (name == F){
     name <- substitute(to_save)
   }
   saveRDS(to_save, file.path(pct_data, region, paste0(name, ".Rds")))
-  
-  # Simplify data checked with before and after using:
-  # plot(l$gendereq_sideath_webtag)
-  to_save@data <- round_df(to_save@data, 5)
-  
-  # Simplify geom
-  geojson_write( ms_simplify(to_save, keep = 0.1), file = file.path(pct_data, region, name))
```

Comment:
Aha that's where the geojson write code was ... reckon we should have used a parameter for the keep value but that's purely academic now. Good thing: will be quicker to write. Potential downside: if lots of people want to download the route network, that will put a lot of work onto our server, non? Suggest because the route network layer and zones will always want to be for the whole region we should still just send a link for those.